installation/install.sh

```
@@ -1094,7 +1094,7 @@ do_check_system(){
 
         # kill zstack if it's still running
         ZSTACK_PID=`ps aux | grep 'appName=zstack' | grep -v 'grep' | awk '{ print $2 }'`
-        [ ! -z $ZSTACK_PID ] && pkill -9 $ZSTACK_PID
+        [ ! -z "$ZSTACK_PID" ] && kill -9 $ZSTACK_PID
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)